packages/lib/survey/utils.ts

```diff
@@ -89,3 +92,15 @@ export const anySurveyHasFilters = (surveys: TSurvey[] | TLegacySurvey[]): boole
     return false;
   });
 };
+
+export const revalidateMembershipByEnvironmentId = async (environmentId: string) => {
```

Comment:
Can we also maybe move this function into the membership utils (not survey utils) as they are connected to the membership model and not the survey model